code/assurance/build.gradle.kts

```diff
@@ -0,0 +1,44 @@
+import com.adobe.marketing.mobile.gradle.BuildConstants
+
+plugins {
+    id("aep-library")
+}
+
+val mavenCoreVersion: String by project
+
+aepLibrary {
+    namespace = "com.adobe.marketing.mobile.assurance"
+    compose = true
+    //enableSpotless = true
+    //enableCheckStyle = true
+
+    publishing {
+        gitRepoName = "aepsdk-assurance-android"
+        addCoreDependency(mavenCoreVersion)
+
+        addMavenDependency("org.jetbrains.kotlin", "kotlin-stdlib-jdk8", BuildConstants.Versions.KOTLIN)
+        addMavenDependency("androidx.appcompat", "appcompat", BuildConstants.Versions.ANDROIDX_APPCOMPAT)
+        addMavenDependency("androidx.compose.runtime", "runtime", BuildConstants.Versions.COMPOSE)
+        addMavenDependency("androidx.compose.material", "material", BuildConstants.Versions.COMPOSE_MATERIAL)
+        addMavenDependency("androidx.compose.animation", "animation", BuildConstants.Versions.COMPOSE)
+        addMavenDependency("androidx.activity", "activity-compose", BuildConstants.Versions.ANDROIDX_ACTIVITY_COMPOSE)
+    }
+}
+
+dependencies {
+    // Stop using SNAPSHOT after Core release.
+    implementation("com.adobe.marketing.mobile:core:$mavenCoreVersion-SNAPSHOT")
+
+    // Compose Navigation
+    implementation("androidx.navigation:navigation-compose:2.4.0")
```

Comment:
Can you also add these as publishing dependencies?

Reply:
Oops! Yes. Added them now.